openreview/venue/matching.py

```diff
@@ -995,7 +995,13 @@ def deploy_sac_assignments(self, assignment_title, overwrite):
                         if overwrite:
                             sac_group.members=[]
                         sac_group.members.append(sac)
-                        client.post_group(sac_group)
+                        client.post_group_edit(
```

Comment:
I think you can post the edit to override the members only:

```
invitation = venue.get_meta_invitation_id(),
readers = [venue.venue_id],
writers = [venue.venue_id],
signatures = [venue.venue_id],
group = openreview.api.Group(
     id = sac_group_id,
     members = [sac]
)
```

or add_members to add a sac. I think we will always override so no need to keep both choices.